syntax = "proto2";
package echo;

option cc_generic_services = true;

message EchoRequest
{
    required int32 id = 1;
    required string msg = 2;
}

message  EchoResponse 
{
    required int32 id = 1;
    required string msg = 2;
}

service EchoService
{
    rpc Echo (EchoRequest) returns (EchoResponse);
}
